### ISSUE TITLE
Remove direct references to `field.(something)` from twig templates

### ISSUE THREAD

**Issue by I-Valchev** [bolt/core#1871]
The `*.html.twig` where `*` is a bolt field should not use `field.` directly.

This is because in several places reuse them by overriding properties.

So if somewhere we use this for example:
```
{% set setPath = field.definition.get('upload')|placeholders({'contenttype': record.contenttype}) %} 
```

It should become:

```
{% if setPath is not defined %}
    {% set setPath = field.definition.get('upload')|placeholders({'contenttype': record.contenttype}) %} 
{% endif %}
```

**Comment by tambait** [comment]
Hi, 
I would like to work on this issue, if that's ok :) I have a question related to the issue (as a general note, I didn't use Bolt before, so don't have big picture), should all Twig variables with `field.*` be modified or there are some exceptions? I see in `core/templates/_partials/fields` there are different variables using `field.*`, for example,   `set limit`, `set variant`, 'set prefix' etc.

**Comment by I-Valchev** [comment]
Hi @tambait, happy to assign this issue to you!

As things stand, I do not thing there should be exceptions. Wrapping the login in `if` means if there is an override, we use that, and otherwise use the `field`.

Let me know if you bump into something here, or reach out on Slack 🙂

**Comment by I-Valchev** [comment]
Hi @tambait , still looking to work on this issue? ;-)

**Comment by tambait** [comment]
Hi @I-Valchev, yes I can try working on it.

**Comment by I-Valchev** [comment]
Awesome! :-) I'll go ahead and assign it to you.

**Comment by bobdenotter** [comment]
Fixed in #2404
